ulib/FStar.ReflexiveTransitiveClosure.fsti

```diff
@@ -0,0 +1,16 @@
+module FStar.ReflexiveTransitiveClosure
+
```

Comment:
Some high-level remarks here about the intended usage of the module would be useful.
You could also point to examples/preorders/Closure.fst for the reader to see it in action.

Reply:
Done.